server/chat-commands.js

```diff
@@ -828,26 +828,26 @@ const commands = {
 		if (!id) return this.parse('/help makechatroom');
 		if (id.length > MAX_CHATROOM_ID_LENGTH) return this.errorReply("The given room title is too long.");
 		// Check if the name already exists as a room or alias
-		if (Rooms.search(id)) return this.errorReply(`The room '${target}' already exists.`);
-		if (!Rooms.global.addChatRoom(target)) return this.errorReply(`An error occurred while trying to create the room '${target}'.`);
+		if (Rooms.search(id)) return this.errorReply(Chat.html`The room '${target}' already exists.`);
```

Comment:
`errorReply` does not take HTML; this will lead to double-escaping.

Reply:
Ah, that goes for `sendReply` as well I take it. Thanks, that had confused me...

Reply:
`sendReply` sends a protocol message. If you do something like ``sendReply(`|html|...`)``, that will need escaping.

Reply:
Right, but none of the cases I touched did that, which is why I reverted all of those changes.